app/src/main/java/com/github/se/polyfit/model/recipe/RecipeInformation.kt

```diff
@@ -0,0 +1,69 @@
+package com.github.se.polyfit.model.recipe
+
+import android.util.Log
+import com.github.se.polyfit.model.ingredient.Ingredient
+import com.github.se.polyfit.model.nutritionalInformation.MeasurementUnit
+
+/** This is a separate class from the Recipe.kt class to reflect the structure of the api */
+data class RecipeInformation(
+    val vegetarian: Boolean,
+    val vegan: Boolean,
+    val glutenFree: Boolean,
+    val dairyFree: Boolean,
+    val listIngredients: List<Ingredient>,
```

Comment:
```suggestion
    val ingredients: List<Ingredient>,
```

Reply:
Yeah you're right that a way better naming scheme.